src/main/java/mat/server/CQLConstantServiceImpl.java

```diff
@@ -205,4 +220,28 @@ private CQLTypeContainer getCQLTypeInformation() {
 	private QDMContainer getQDMInformation() {
 		return QDMUtil.getQDMContainer();
 	}
+
+	public void getFhirAttributeAndDataTypes(CQLConstantContainer cqlConstantContainer) {
+		List<String> fhirAttributeList = new ArrayList<>();
+		List<String> fhirDataTypeList = new ArrayList<>();
+
+		try {
+			JSONArray fhirResponse = new JSONArray(restTemplate.getForObject(fhirMatServicesUrl + FHIR_MAT_SERVICES_RECOURSE_FOR_ATTRIBUTES, String.class));
+
+			for (int i = 0; i < fhirResponse.length(); i++) {
+
+				if(!fhirDataTypeList.contains(fhirResponse.getJSONObject(i).get("matDataTypeDescription").toString())) {
```

Comment:
I would suggest to introduce a pojo with following fields: 
- matDataTypeDescription
- matDataTypeDescription
- matAttributeName